bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/provider/AbstractResourceBundleProvider.java

```diff
@@ -306,15 +306,15 @@ protected void putNewPortfolio(Vendor vendor, List<String> portfolio) {
      */
     protected String getParserType(URL url) {
         String fileName = url.getPath();
-        int fileExtesionStartIndex = fileName.lastIndexOf(".") + 1;
-        if (fileExtesionStartIndex == -1) {
+        int fileExtensionStartIndex = fileName.lastIndexOf(".") + 1;
+        if (fileExtensionStartIndex == -1) {
```

Comment:
This looks like a bug. How can `fileExtensionStartIndex` equals to -1 when we add 1 in the line before?

We should either do this:

```suggestion
        if (fileName.lastIndexOf(".") == -1) {
```

Or add one later in `fileName.substring(fileExtensionStartIndex + 1);

Maybe do it similar to `AbstractFileProvider#getParserType()`.

Reply:
I think you're right so it's fixed now. :+1: